redbot/cogs/warnings/warnings.py

```diff
@@ -561,6 +561,36 @@ async def warnings(self, ctx: commands.Context, member: Union[discord.Member, in
                     ),
                 )
 
+    @warnings.command(name="guild")
+    @commands.guild_only()
+    @checks.admin()
+    async def warnings_guild(self, ctx: commands.Context):
+        """List all members with warnings in this guild."""
```

Comment:
For end user UI experience this should probably be renamed to "server", and aliased to "guild" as a command name.